commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java

```diff
@@ -627,6 +663,29 @@ public String getOrderByType() {
 			"orderByType", "desc");
 	}
 
+	public OrderItemFilterImpl getOrderItemFilter() throws PortalException {
+		OrderItemFilterImpl orderItemFilter = new OrderItemFilterImpl();
+
+		orderItemFilter.setAdvancedSearch(
+			ParamUtil.getBoolean(
+				_commerceOrderRequestHelper.getRequest(), "advancedSearch"));
+		orderItemFilter.setAndOperator(
+			ParamUtil.getBoolean(
+				_commerceOrderRequestHelper.getRequest(), "andOperator", true));
+		orderItemFilter.setSku(
+			ParamUtil.getString(
+				_commerceOrderRequestHelper.getRequest(), "sku"));
+		orderItemFilter.setName(
+			ParamUtil.getString(
+				_commerceOrderRequestHelper.getRequest(), "name"));
+
+		orderItemFilter.setKeywords(
+			ParamUtil.getString(
+				_commerceOrderRequestHelper.getRequest(), "keywords"));
```

Comment:
Let's alphabetize these setters, and no blank lines are necessary in between them.